gradle.properties

```diff
@@ -2,11 +2,13 @@
 org.gradle.jvmargs=-Xmx1G
 # Fabric Properties
 # check these on https://modmuss50.me/fabric.html
-minecraft_version=1.20.1
-yarn_mappings=1.20.1+build.1
-loader_version=0.14.21
+minecraft_version=1.20.2
+yarn_mappings=1.20.2+build.4
+loader_version=0.14.23
+#Fabric api
+fabric_version=0.90.0+1.20.2
```

Comment:
How come this was added? NRS doesn't depend on the Fabric API, it also isn't added to the build.gradle.kts file.

Reply:
I don't know to be honest, just copied it from the fabric's site :)

Reply:
Okie dokes, would there be any issue removing it since it's redundant?

Reply:
go ahead